packages/README.md

```diff
@@ -0,0 +1,14 @@
+This directory contains the npm packages hosted by the react-native-windows repo. 
+
+## Scoped Packages
+
+Several packages are organized into specific package scopes:
+
+- `@react-native`: Copies of private upstream packages already using the `@react-native` scope. E.g. `@react-native/tester` for the RNTester package.
+- `@rnw-scripts`: Scripts for repository tooling. Published as part of the master branch.
+- `@react-native-windows`: Internal packages used by react-native-windows itself. Versioned and published alongside RNW.
```

Comment:
Bots shouldn't go in here, even if they're here right now. Going to make a separate scope for them.